src/week1/calculator/verify.go

```diff
@@ -0,0 +1,13 @@
+package calculator
+
+import "regexp"
+
+var (
+	regex, _ = regexp.Compile(`^\-?\d+.?\d*\ +(\+|\-|\*|\/){1}\ \-?\d+.?\d*$`)
```

Comment:
- `{1}` seems unnecessary
- `1. + 12.` will be passed by this regex
- multiple spaces support would be great

Reply:
- I think `1. + 12. ` is okay :D
- `\ +` support multiple spaces